coreos-base/oem-ec2-compat/files/grub-ec2.cfg

```diff
@@ -5,3 +5,7 @@ set oem_id="ec2"
 # Blacklist the Xen framebuffer module so it doesn't get loaded at boot
 # Disable `ens3` style names, so eth0 is used for both ixgbevf or xen.
 set linux_append="modprobe.blacklist=xen_fbfront net.ifnames=0"
+
+if [ "$grub_platform" = pc ]; then
+	set linux_console="console=ttyS0,115200n8"
```

Comment:
Add grub serial so you see it's output too:

          serial com0 --speed=115200 --word=8 --parity=no
￼          terminal_input console serial_com0
￼          terminal_output console serial_com0